apps/dashboard/src/components/primitives/tag-input.tsx

```diff
@@ -79,10 +122,18 @@ const TagInput = forwardRef<HTMLInputElement, TagInputProps>((props, ref) => {
             </button>
           </Badge>
         ))}
+
+        {showAddButton && !isFocussed && (
+          <Badge variant="outline" kind="tag">
+            <button type="button" className="rounded-full outline-none focus:outline-none">
```

Comment:
please update the button size as it looks wider than in the designs
![Screenshot 2024-10-22 at 00 29 28](https://github.com/user-attachments/assets/0d7e9b78-dc44-494f-b957-b04ad6134c5e)